.travis.yml

```diff
@@ -12,7 +12,11 @@ jobs:
       before_script:
         - pip install -q flake8
       script:
+<<<<<<< HEAD
+        - flake8
+=======
         - flake8 --exclude=doorknob/__init__.py,migrations
+>>>>>>> c7b73082a79eef302b7168ff3e80dc3c2293398b
```

Comment:
@uyennguyen16900 , it looks like you changed the command for ```flake8``` on your branch. Was the previous command (the one that says ```- flake8 --exclude=doorknob/__init__.py,migrations``` giving you errors?

In it's current state these 5 lines will cause a merge conflict, so if there are no errors using the 2nd command I think we should choose that over the basic ```- flake8```, as it skips over certain files during testing that we don't need to worry about.